_aggregations/bucket/geohash-grid.md

```diff
@@ -255,7 +255,7 @@ Geohash grid aggregation requests support the following parameters.
 Parameter | Data type | Description
 :--- | :--- | :---
 field | String | The field on which aggregation is performed. This field must be mapped as a `geo_point` or `geo_shape` field. If the field contains an array, all array values are aggregated. Required.
-precision | Integer | The zoom level used to determine grid cells for bucketing results. Valid values are in the [0, 15] range. Optional. Default is 5. 
+precision | Integer | The granularity level used to determine grid cells for bucketing results. Valid values are in the [0, 12] range. Optional. Default is 5. 
```

Comment:
Should we say what about the grid cells is being determined?

Reply:
Added more info.